src/main/java/io/codiga/plugins/jetbrains/model/CodingAssistantVariableContext.java

```diff
@@ -0,0 +1,12 @@
+package io.codiga.plugins.jetbrains.model;
+
+import static io.codiga.plugins.jetbrains.Constants.GET_FILENAME;
+import static io.codiga.plugins.jetbrains.Constants.GET_FILENAME_NO_EXT;
+
+public class CodingAssistantVariableContext {
```

Comment:
The context should have everything you need to do the transformation.

So something like 

```
public class CodingAssistantContext {
   private VirtualFile virtualFile;
   public CodingAssistantContext(VirtualFile vf) {
      this.virtualFile = vf;
   }

}


```